typescript/mint-specific-nft/pages/api/getNfts.ts

```diff
@@ -0,0 +1,95 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import fs from 'fs';
+import { ThirdwebSDK } from '@thirdweb-dev/sdk';
+import { ethers } from 'ethers';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+export interface NFT {
+  id: number;
+  name: string;
+  description: string;
+  url: string;
+  price: number;
+  minted: boolean;
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse
+) {
+  // Read and format the NFT data
+  const rawData = fs.readFileSync(
```

Comment:
why do you need to do this? can't you just import the JSON directly in the imports?